app/js/services/individuService.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('ddsCommon').service('IndividuService', function($filter, situationsPro, nationalites) {
-    var statutsSpecifiques = _.filter(situationsPro, 'isStatutSpecifique');
+    var statutsSpecifiques = situationsPro;
```

Comment:
Ok, I removed the intermediate variable and did some renaming, but there was no obvious simplification about the `find`.